### PR TITLE
Get all non-local tracks from a spotify playlist

### PR DESCRIPTION
**Pull request by Omglolyes**

### CODE REVIEW

**Comment by stephanebruckert** [comment]
That's nice, maybe just rename the filename to `examples/playlist_all_non_local_tracks.py`?

File: examples/playlist_all_tracks.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+tracks.extend(result['items'])
+
+# if playlist is larger than 100 song, continue loading it until end
+while result['next']:
+	result = sp.next(result)
+	tracks.extend(result['items'])
+
+# remove all local songs
+for item in tracks:
+	if item["is_local"] == True:
```
Comment (stephanebruckert) [inline examples/playlist_all_tracks.py:20]: no need for `== True`

File: examples/playlist_all_tracks.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+		tracks.remove(item)
+		
+return tracks
```
Comment (stephanebruckert) [inline examples/playlist_all_tracks.py:23]: not sure this is useful, do you want `pp(tracks)` instead?

File: examples/playlist_all_tracks.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# get all non-local tracks of a playlist
+from spotipy.oauth2 import SpotifyClientCredentials
+import spotipy
+from pprint import pprint as pp
+
+sp = spotipy.Spotify(client_credentials_manager=SpotifyClientCredentials())
```
Comment (stephanebruckert) [inline examples/playlist_all_tracks.py:6]: afaik removing tracks from playlists needs user authentication, did you try this code?

File: examples/playlist_all_tracks.py (hunk 1 above)
Comment (Omglolyes) [reply examples/playlist_all_tracks.py:20]: it `if item["is_local"]` should be enough?

Comment (stephanebruckert) [reply examples/playlist_all_tracks.py]: Yes

**Comment by Omglolyes** [comment]
i don't know if printing the result really gives any insight into what it does. especially pprint throws way to much information around. maybe only print track name and if it is local or not?

**Comment by stephanebruckert** [comment]
You could print a count of the total local tracks deleted? 

> just deleted __ local track(s) in playlist ____

With the playlist you chose it might be 0, which is fine as people will at least have an example and will be able to replace the playlist id with their own

**Pull request merged**